packit/cli/dist_git_init.py

```diff
@@ -107,6 +107,15 @@
     help="Comma separated list of target branches in dist-git to release into. "
     "(defaults to rawhide)",
 )
+@click.option(
+    "--dist-git-branches-mapping",
+    help="""
+    JSON dictionary of target branches in dist-git to release into
+for which `fast_forward_merge_into` syntax will be used, e.g. '{"fedora-rawhide": ["f39", "f40"]}'.
+If not provided and --dist-git-branches is not provided as well,
+defaults to '{"fedora-rawhide": ["fedora-stable"]}').
```

Comment:
Is this a good default? Wouldn't `{"fedora-rawhide": ["fedora-branched"]}` be better? For example right now `fedora-stable` doesn't include `f41`.

Reply:
that's a good point, we have it like this also in docs, I will change it :+1: